src/mocks/company/resolvers.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpResponse, type HttpResponseResolver } from 'msw';
+
+import type { CompanyListResponse } from '@/mocks/company/schemas.ts';
+
+type companyResolver = {
+  companyList: HttpResponseResolver<never, never, CompanyListResponse[]>;
+};
+
+export const companyResolver: companyResolver = {
```

Comment:
마이너한데 타입 이름은 PascalCase 로 작성해야 합니다!

```suggestion
type CompanyResolver = {
  companyList: HttpResponseResolver<never, never, CompanyListResponse[]>;
};

export const companyResolver: CompanyResolver = {
```

Reply:
감사합니다~